eng/azure-pipelines.yml

```diff
@@ -70,7 +70,7 @@ stages:
             demands: ImageOverride -equals build.ubuntu.2204.amd64
         container:
           # when updating, change emscripten version here: ../.devcontainer/emscripten-version.txt
```

Comment:
Does this new image tag use the same emscripten version?

Reply:
It doesn't have Emscripten in it at all. This is leftover cruft.